controller/employeeController.ts

```diff
@@ -13,6 +13,7 @@ module.exports = function (app: Application) {
             console.error(e);
         }
 
-        res.render('list-delivery-employees', { deliveryEmployees: data })
+        const { token } = req.session;
```

Comment:
any reason for this?

Reply:
We are checking the token value to hide links based on the value of token in templates/master